index.js

```diff
@@ -158,7 +161,7 @@ const transformDom = (dom) => {
                     newData = {
                         data: {},
                         content,
-                        nodeType: htmlAttrs[type][name],
+                        nodeType: invalidNodeTypes.includes(htmlAttrs[type][name]) ? "paragraph" : htmlAttrs[type][name],
```

Comment:
sorry, I tried run locally to see if I understand it better, but I couldn't yet 😜 

in which case do we get to this line? :) can we add a test for that? From what I understood, this 

```
htmlTest(
    '<em>Test</em>',
    '<i>Test</i>'
);
```

is passing due to the addition of `case 'em':`, right?